### PR TITLE
docs: confluence document migration

### PR DESCRIPTION
**Pull request by curlyreeve**

### CODE REVIEW

File: Images/image-20240125-060057.png
Comment (Justus-at-Tazama) [inline Images/image-20240125-060057.png:1]: Folder should be lower-case which is why it's not rendering in the readme file.

**Review by Justus-at-Tazama — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,21 +1,31 @@
+r# Rule Command Line Interface Documentation
```
Comment (Justus-at-Tazama) [inline README.md:1]: The heading # is prefaced with an errant 'r' which is why the heading doesn't render properly

**Review by Justus-at-Tazama — CHANGES_REQUESTED** [review]

**Review by Sandy-at-Tazama — CHANGES_REQUESTED** [review]
Image does not render

**Review by Sandy-at-Tazama — CHANGES_REQUESTED** [review]
This images folder is capitalised `Images`

**Review by RobReeveLexTego — DISMISSED** [review]

**Review by Sandy-at-Tazama — DISMISSED** [review]

**Review by RobReeveLexTego — DISMISSED** [review]
image now in images, and not Images
file now called readme.md and not README.md

**Comment by Sandy-at-Tazama** [comment]
@RobReeveLexTego  File still called README.md when I checked it now?

**Comment by RobReeveLexTego** [comment]
image
Note the commit @Sandy-at-Tazama  - this is one of those horrible windows / linux issues

**Comment by Justus-at-Tazama** [comment]
> image now in images, and not Images file now called readme.md and not README.md

Hi,

The all-caps README.md is the standard for GitHub and all of our repositories. Please retain the all-caps name?

**Comment by curlyreeve** [comment]
Updated and done

**Review by Sandy-at-Tazama — APPROVED** [review]

**Review by Justus-at-Tazama — APPROVED** [review]

**Pull request merged**